tests/garage/misc/test_wrappers.py

```diff
@@ -0,0 +1,20 @@
+import gym
+
+from garage.misc.wrappers import ResizeAndGrayscaleWrapper, StackFramesWrapper
+from tests.fixtures import TfGraphTestCase
```

Comment:
These classes have nothing to do with TensorFlow so they shouldn't be using this test fixture